### ISSUE TITLE
Manually upgrade flutter

### ISSUE THREAD

**Issue by cal2dev** [flutter/flutter#21417]
## How to Manually upgrade Flutter

Hi I am trying to upgrade flutter by running
`flutter upgrade`
and tried
`flutter doctor`
in both cases my antivirus blocked 7za.exe

```
Access is denied.
 Command: C:\flutter\bin\cache\dart-sdk\lib\_internal\pub\asset\7zip\7za.exe e
data.tar.gz 

```
Please share the details how we can manually upgrade flutter.

**Comment by zoechi** [comment]
What antivirus software are you using?

**Comment by cal2dev** [comment]
**Mcafee Endpoint Security**
I dont know its a by default settings to block or a  special setting in my pc only but this scenario can come to some one else also, Will be better if we know the steps of manually to updating flutter.

**Comment by cal2dev** [comment]
Any help please...

**Comment by cal2dev** [comment]
7za.exe is causing issue in downloading packages also [check this issue](https://github.com/flutter/flutter/issues/23058)

**Comment by zoechi** [comment]
You can just use 
```
git clone https://github.com/flutter/flutter.git
# add flutter/bin to PATH
flutter doctor
```

**Comment by cal2dev** [comment]
yes for the work around I did this and everything start working but again I got issue due to 7za.exe.
Please [check this issue](https://github.com/flutter/flutter/issues/23058).

I am not able to install packages. Please help any work around for this.
